.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I can't see it from Git, but be sure to run `chmod +x` on this and the `pre-commit` file in a Linux environment (ala WSL) because otherwise WSL/MacOS/actual Linux will complain about this not being executable. We've had that happen in Sapphire a while ago and I ended up running a bash script to add the permission to all the relevant files.

Reply:
I created those, as per husky docs, with the husky CLI to alleviate those issues, but I will double check.